app/views/curation_concerns/base/_attribute_rows.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= presenter.attribute_to_html(:creator, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:contributor, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:college, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:department, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:admin_set, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:contributor) %>
+<%= presenter.attribute_to_html(:college) %>
```

Comment:
I think we'd want to keep the faceted label on this and department - keep these as links from the show page?